wps/adapters/outbound/http.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Outbound HTTP calls """
+
+import httpx
+from pydantic import BaseSettings, Field
+
+from wps.ports.outbound.http import AccessCheckPort
+
+__all__ = ["AccessCheckConfig", "AccessCheckAdapter"]
+
+TIMEOUT = 60
+
+
+class AccessCheckConfig(BaseSettings):
+    """Config parameters for checking dataset access."""
+
+    download_access_url: str = Field(
+        ...,
+        example="http://127.0.0.1/download_access",
+        description="URL pointing to the internal download access API"
+        "provided by the claims repository.",
```

Comment:
This is a configuration detail, might come from somewhere else.
```suggestion
        description="URL pointing to the internal download access API.",
```